packages/next-urql/examples/1-with-urql-client/pages/index.tsx

```diff
@@ -28,12 +26,11 @@ Home.getInitialProps = () => {
 
 export default withUrqlClient((_ssr: object, ctx?: NextUrqlPageContext) => {
   return {
-    url: 'https://graphql-pokemon.now.sh',
+    url: 'https://graphql-pokemon2.vercel.app',
```

Comment:
The new API is already live (but not OSS for final just yet)

```suggestion
    url: 'https://trygql.dev/graphql/basic-pokedex',
```